packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -16,13 +16,19 @@ if (process.env.FIREBASE_APP_DIRECTORY) {
   projectRoot = projectRoot.concat("/", process.env.FIREBASE_APP_DIRECTORY);
 }
 
-// Determine which build runner to use
-let cmd = DEFAULT_COMMAND;
-if (process.env.MONOREPO_COMMAND) {
-  cmd = process.env.MONOREPO_COMMAND;
+// Determine which command to run the build
+const cmd = process.env.MONOREPO_COMMAND || DEFAULT_COMMAND;
+
+// Parse args to pass to the build command
+let cmdArgs: string[] = [];
+if (process.env.MONOREPO_BUILD_ARGS) {
```

Comment:
Where is MONOREPO build args set? I dont see it anywhere in the buildpacks code either

Reply:
MONOREPO_BUILD_ARGS is a new environment variable added in cl/631875763

Reply:
Maybe I'm missing something, but I can't seem to find MONOREPO_BUILD_ARGS in that cl. Could you help point me to it?

Reply:
Whoops sorry, just uploaded the changes - should be in line 95 of `firebasenx/main.go` now